### ISSUE TITLE
Learning Linear Regression

### ISSUE THREAD

**Issue by MidasXIV** [MidasXIV/Artificial-Intelliegence--Deep-Learning--Tensor-Flow#3]
| Source | Description | Status | Time | Colab |
|:-------- | :------------- | :-----: | :------ | :-----: |
|<img src="http://www.stickpng.com/assets/images/580b57fcd9996e24bc43c545.png" alt="Video" width="125"/>| [Tensorflow Basic - Linear Regression](https://www.youtube.com/watch?v=Xiab2JhwzYY)<h6>Video By Mark Jay<br>Custom loss and optimizer Functions<br>Does not use Keras</h6> | 20 minutes | Done | [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/MidasXIV/Artificial-Intelliegence--Deep-Learning--Tensor-Flow/blob/master/1.%20Linear%20Regression/TF_without_Keras_Linear_Regression.ipynb) |
|<img src="http://www.stickpng.com/assets/images/580b57fcd9996e24bc43c545.png" alt="Video" width="125"/>| [Regression Neural Networks for Keras and TensorFlow](https://www.youtube.com/watch?v=uLa-b3JxWAM)<h6>Video By Jeff Heaton<br>Good explanation of using plotlib<br>Cars and fuel consumption Dataset used </h6> | 10 minutes | To Do | [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/MidasXIV/Artificial-Intelliegence--Deep-Learning--Tensor-Flow/blob/master/1.%20Linear%20Regression/TF_without_Keras_Linear_Regression.ipynb) |
|<img src="http://www.stickpng.com/assets/images/580b57fcd9996e24bc43c545.png" alt="Video" width="125"/>| [Neural Networks Regression Models](https://www.youtube.com/watch?v=WskWc15bcy4)<h6>Video By Jeff Heaton<br>Scikit-Learn + Keras<br>Custom Dataset of Youtube videos </h6> | 20 minutes | To Do | [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/MidasXIV/Artificial-Intelliegence--Deep-Learning--Tensor-Flow/blob/master/1.%20Linear%20Regression/TF_without_Keras_Linear_Regression.ipynb) |
